lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -256,11 +259,18 @@ def _get_user_query(self, parameters: JSON, terms: List[JSON]) -> JSON:
                 }
             )
             self.logger.debug("QUERY: {{}} default: {}", filter)
+            return {"bool": {"filter": filter}}
         else:
             # Either "user" was specified and is already added to the filter,
```

Comment:
s/is already/will be/ ?

Reply:
Right; because it used to be added first, but in order to ensure that user/access terms were in a consistent order for unit testing I introduced the local term variables which are added to the filter at the end. Sigh. No good (or ugly but necessary) deed goes unpunished...